backend/app/main.py

```diff
@@ -53,3 +56,42 @@ async def db_is_alive(session: Session = Depends(get_session)):
         raise HTTPException(
             status_code=status.HTTP_503_SERVICE_UNAVAILABLE, detail="DB is unreachable"
         )
+
+
+@app.post("/create_document")
+async def create_document(session: Session = Depends(get_session)):
+    doc = Document()
+    session.add(doc)
+    session.commit()
+    session.refresh(doc)
+    document_id = doc.document_id
+    # Also create the partition in one go.
+    session.execute(
+        text(
+            f"""
+            CREATE TABLE assignments_{document_id} PARTITION OF assignments
+            VALUES IN ('{document_id}')
+        """
+        )
+    )
+    return doc
+
+
+@app.post("/update_assignments")
```

Comment:
I: PATCH rather than POST since we're doing partial updates and (TBD) update document from `document_id` with `Assignments` as just `(geoid, zone)` tuples? 
```suggestion
@app.patch("/update_assignments/{document_id}") # TBD will we need a response model?
```
See [examples in docs](https://fastapi.tiangolo.com/tutorial/body-updates/) in case helpful!